### PR TITLE
fix(mirrormaker): missing omitempty in replication flow

### PR DESCRIPTION
**Pull request by Serpentiel**
fixes missing `omitempty` in replication flow

### CODE REVIEW

File: mirrormaker_replication_flow.go [hunk 1]
```diff
@@ -11,17 +11,17 @@ type (
 
 	// ReplicationFlow a replication flow entity
 	ReplicationFlow struct {
-		Enabled                         bool     `json:"enabled"`
+		Enabled                         bool     `json:"enabled,omitempty"`
 		OffsetSyncsTopicLocation        string   `json:"offset_syncs_topic_location,omitempty"`
 		SourceCluster                   string   `json:"source_cluster,omitempty"`
 		TargetCluster                   string   `json:"target_cluster,omitempty"`
```
Comment (byashimov) [inline mirrormaker_replication_flow.go:17]: These two fields are required https://api.aiven.io/doc/#tag/Service:_Kafka_MirrorMaker/operation/ServiceKafkaMirrorMakerCreateReplicationFlow

Comment (byashimov) [reply mirrormaker_replication_flow.go:17]: Or it used as all-in-one object?

Comment (Serpentiel) [reply mirrormaker_replication_flow.go:17]: yes, it's used in both request and response

**Review by byashimov — APPROVED** [review]
Lgtm

**Pull request merged**